src/qt/pivx/dashboardwidget.cpp

```diff
@@ -874,6 +869,23 @@ void DashboardWidget::windowResizeEvent(QResizeEvent* event)
 void DashboardWidget::onHideChartsChanged(bool fHide)
 {
     fShowCharts = !fHide;
+
+    if (fShowCharts && !stakesFilter) {
+        // chart filter
+        stakesFilter = new TransactionFilterProxy(this);
+        stakesFilter->setDynamicSortFilter(false);
+        stakesFilter->setSortCaseSensitivity(Qt::CaseInsensitive);
+        stakesFilter->setFilterCaseSensitivity(Qt::CaseInsensitive);
+        stakesFilter->setOnlyStakes(true);
+        stakesFilter->setSourceModel(txModel);
+        hasStakes = stakesFilter->rowCount() > 0;
+    } else {
+        if (stakesFilter) {
+            stakesFilter->setSourceModel(nullptr);
+            stakesFilter->deleteLater();
```

Comment:
These lines are causing a consistent segfault here.
Couple times the chain database was also corrupted for some reason.